### PR TITLE
Add force restore option

### PR DESCRIPTION
**Pull request by gtrancillo**

### CODE REVIEW

File: scripts/restore.sh [hunk 1]
```diff
@@ -273,9 +279,13 @@ function restore() {
     check_empty_input
     check_data_dir_exist
 
-    color yellow "Restore will overwrite the existing files, continue? (y/N)"
-    read -p "(Default: n): " READ_RESTORE_CONTINUE
-    if [[ $(echo "${READ_RESTORE_CONTINUE:-n}" | tr [a-z] [A-Z]) == "Y" ]]; then
-        restore_file
-    fi
+    if $FORCE_RESTORE; then
+      restore_file
```
Comment (ttionya) [inline scripts/restore.sh:283]: Please keep the correct indentation (4 spaces).

File: scripts/restore.sh [hunk 2]
```diff
@@ -262,6 +263,11 @@ function restore() {
                 RESTORE_FILE_SENDS="$(basename "$1")"
                 shift
                 ;;
+              --force-restore)
```
Comment (ttionya) [inline scripts/restore.sh:266]: Please keep the correct indentation (4 spaces).

File: Dockerfile [hunk 3]
```diff
@@ -12,7 +12,6 @@ FROM rclone/rclone:1.59.0 AS image-linux-armv7
 
 # for linux/arm/v6 platform
 FROM alpine:3.15 AS image-linux-armv6
-
```
Comment (ttionya) [inline Dockerfile:15]: Don't delete blank lines, keep them to make it easier to read.

**Review by ttionya — CHANGES_REQUESTED** [review]
@gtrancillo , only a few typographical errors need to be corrected.

File: Dockerfile (hunk 3 above)
Comment (gtrancillo) [reply Dockerfile:15]: I put it together because those two lines are needed for ARMv6 platform.

**Pull request merged**